buildconfig/pygame-stubs/constants.pyi

```diff
@@ -20,6 +20,12 @@ ANYFORMAT: int
 APPACTIVE: int
 APPFOCUSMOUSE: int
 APPINPUTFOCUS: int
+APP_DIDENTERBACKGROUND: int
+APP_DIDENTERFOREGROUND: int
+APP_LOWMEMORY: int
+APP_TERMINATING: int
+APP_WILLENTERBACKGROUND: int
+APP_WILLENTERFOREGROUND: int
```

Comment:
The previous `APP` events don't have underscores between `APP` and `TYPE`. Readability vs consistency?

The previous ones also don't seem to be documented anywhere.

Reply:
The new ones have underscore because the respective SDL constants have them. not sure whats up with the older `APP` constants

Reply:
What's the consistent way here? We should probably change them to that.

Reply:
So far, pygame has always followed SDL constants directly, usually only stripping the initial `SDL_` before exporting an SDL constant. SDL initially did not use `_`s in the event constant names, but looks like nowadays, in newer constants they add, they seem to use `_`s more often. The easiest way for pygame would be to stick to SDL event names for consistency, also IMHO the names with underscores are more readable, but older constants added previously do not make use of them

Reply:
Hrmm, I can't find `SDL_APPACTIVE` inside SDL (nor SDL_APPINPUTFOCUS... didn't look for the others). What am I making a mistake with?

Reply:
Also tried SDL_APP_ACTIVE, and SDL_APP_INPUTFOCUS

Reply:
They are defined for compatibility with SDL1 `ACTIVEEVENT`, the enum is defined like 
```c
typedef enum {
    SDL_APPFOCUSMOUSE,
    SDL_APPINPUTFOCUS,
    SDL_APPACTIVE
} PygameAppCode;
```
These are not events, but values that `event.state` can take when event is of type `ACTIVEEVENT`